specification/eventhub/resource-manager/Microsoft.EventHub/preview/2021-12-01-preview/examples/EventHubs/EHEventHubCreate.json

```diff
@@ -0,0 +1,130 @@
+{
+  "parameters": {
+    "namespaceName": "sdk-Namespace-5357",
+    "resourceGroupName": "Default-NotificationHubs-AustraliaEast",
+    "eventHubName": "sdk-EventHub-6547",
+    "api-version": "2021-12-01-preview",
+    "subscriptionId": "5f750a97-50d9-4e36-8081-c9ee4c0210d4",
+    "parameters": {
+      "properties": {
+        "messageRetentionInDays": 4,
+        "partitionCount": 4,
+        "status": "Active",
+        "captureDescription": {
+          "enabled": true,
+          "encoding": "Avro",
+          "intervalInSeconds": 120,
+          "sizeLimitInBytes": 10485763,
+          "destination": {
+            "name": "EventHubArchive.AzureBlockBlob",
+            "properties": {
+              "storageAccountResourceId": "/subscriptions/e2f361f0-3b27-4503-a9cc-21cfba380093/resourceGroups/Default-Storage-SouthCentralUS/providers/Microsoft.ClassicStorage/storageAccounts/arjunteststorage",
+              "blobContainer": "container",
+              "archiveNameFormat": "{Namespace}/{EventHub}/{PartitionId}/{Year}/{Month}/{Day}/{Hour}/{Minute}/{Second}"
+            }
+          }
+        },
+        "throttleRules": [
+          {
+            "ruleName": "Ingress1",
+            "targetId": "AppId=Guid1",
+            "metricId": "IncomingBytes",
+            "throttleLimit": 137429547
+          },
+          {
+            "ruleName": "Egress1",
+            "targetId": "ConsumerGroupName=AmqpGroup1",
+            "metricId": "OutgoingBytes",
+            "throttleLimit": 3951729
+          },
+          {
+            "ruleName": "Egress2",
+            "targetId": "SASKey=SASKeyName",
+            "metricId": "OutgoingMessages",
+            "throttleLimit": 6967
+          },
+          {
+            "ruleName": "Egress3",
+            "targetId": "KafkaClientId=KafkaClientId",
+            "metricId": "OutgoingBytes",
+            "throttleLimit": 3951729
+          },
+          {
+            "ruleName": "Ingress2",
+            "targetId": "SASKey=SASKeyName",
+            "metricId": "IncomingMessages",
+            "throttleLimit": 3456
+          }
+        ]
+      }
+    },
+    "responses": {
```

Comment:
`responses` should move to root level to fix `model validation` errors.

Reply:
Addressed and that helped to resolve the problem. Thanks!